postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -576,9 +585,30 @@ def _run_explain_safe(self, dbname, statement, obfuscated_statement, query_signa
 
         try:
             return self._run_explain(dbname, statement, obfuscated_statement), None, None
+        except psycopg2.errors.UndefinedParameter as e:
+            self._log.debug(
+                "Unable to collect execution plan, clients using the extended query protocol can't be "
+                "explained due to the separation of the parsed query and raw bind parameters: %s",
+                repr(e),
+            )
+            error_response = None, DBExplainError.extended_query_protocol, '{}'.format(type(e))
+            self._explain_errors_cache[query_signature] = error_response
+            self._emit_run_explain_error(dbname, DBExplainError.extended_query_protocol, e)
+            return error_response
+        except psycopg2.errors.UndefinedTable as e:
+            self._log.debug(
+                "Failed to collect execution plan, the query executed may be in a different search_path. "
+                "Alter the provided user's search_path to include the schema: %s",
```

Comment:
This comment is probably more confusing than helpful. Do we have docs for this?

Reply:
The plan is to add documentation for this, we could improve this by making it a check warning and link to the documentation. If a check warning would be too noisy we could leave it as a debug log.